client_ui/package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "clientUi",
   "version": "0.0.0",
-  "dependencies": {},
+  "dependencies": {
+    "osc-client": "^1.1.0"
```

Comment:
According to https://github.com/BublTechnology/ScarletTests/pull/4#issuecomment-149949908 this likely won't cut it since there's no system bring it into the browser.